server/src/db/migrations/20201130113516_create_experiences_table.cjs

```diff
@@ -0,0 +1,33 @@
+const tableName = "experiences";
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  const tableExists = await knex.schema.hasTable(tableName);
+
+  if (!tableExists) {
+    console.log(`Creating ${tableName}`)
+    return knex.schema.createTable(tableName, (table) => {
+      table.bigIncrements("id");
+      table.integer("elevatorId").notNullable();
```

Comment:
we want to use `references` here to indicate a foreign key relationship

Reply:
Story created